File: Java/Basics_java_programs/Basic programs/Odd_even.java
import java.util.Scanner;

public class Odd_even {
    public static void main(String[] args) {
        
        Scanner sc = new Scanner(System.in);

        System.out.println("Enter any number: ");
        
        int a = sc.nextInt();
        sc.close();

        if (a % 2 == 0) {

            System.out.println("Given number is even!");
        }
        else {

            System.out.println("Given number is Odd!");
        }
    }
}

File: oddeven.c
#include <stdio.h>
int main ()
{
    int num;
    printf("Enter a number");
    scanf("%d", &num);
    

    if(num % 2 == 0)
        printf("This number is even");
    else
        printf("This number is odd");


      return 0;  

}